### PR TITLE
Add a note to install-w-machine with pointers if hosted discovery is down

### PR DESCRIPTION
**Pull request by ahaurw01**
Unsticks beginners, fix https://github.com/docker/swarm/issues/1503.

### CODE REVIEW

**Comment by ahaurw01** [comment]
```
The command "go test -v -race ./..." exited with 1.
...
Discovery service returned 500 HTTP status code
```

:laughing: :trollface:

**Comment by abronan** [comment]
ping @moxiegirl (this note might be useful for docker-machine users)

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ Daemon running on each node. Other discovery service backends such as
 	The `swarm create` command returned the  `fe0cc96a72cf04dba8c1c4aa79536ec3`
 	token.
 
+	**Note:** This command relies on Docker Swarm's hosted discovery service. If
```
Comment (moxiegirl) [inline docs/install-w-machine.md]: @ahaurw01 Thanks for the contribution. The Note is formatted incorrectly.  Can you fix that please

```markdown
>**Note**: This command relies on Docker Swarm's hosted discovery service. If
this service is having issues, this command may fail. In this case, see
information on using other types of [discovery backends](./discovery). Check
the [status page](http://status.docker.com/) for service availability.
```

**Comment by vieux** [comment]
@ahaurw01 can you please update to use @moxiegirl's comment

**Comment by amitshukla** [comment]
@ahaurw01 ping - thanks!

**Comment by ahaurw01** [comment]
Sorry, this fell off my radar. I'll update this weekend. Thanks for the reminder @amitshukla.

**Comment by ahaurw01** [comment]
Updated.

**Comment by abronan** [comment]
Thank you @ahaurw01 

LGTM

/cc @moxiegirl @vieux

**Comment by moxiegirl** [comment]
LGTM

**Comment by ahaurw01** [comment]
Thanks!

**Pull request merged**